TMessagesProj/src/main/java/org/telegram/ui/ChatActivity.java

```diff
@@ -2135,30 +2135,8 @@ public void onClick(DialogInterface dialogInterface, int i) {
     }
 
     private boolean searchForHttpInText(CharSequence string) {
```

Comment:
You could also rename `string` to `sequence` or some other more accurate name, while you're on it.
`string.toString()` looks kinda weird…